specs/analytics_v1_pageviews.yaml

```diff
@@ -0,0 +1,291 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: Analytics Pageviews API
+  description: Analytics Pageviews API.
+  termsofservice: https://github.com/wikimedia/restbase#restbase
+  contact:
+    name: Analytics
+    email: analytics@lists.wikimedia.org
+    url: https://www.mediawiki.org/wiki/Analytics
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+x-default-params:
+  domain: analytics.wikimedia.org
+paths:
+  /{module:pageviews}/:
+    get:
+      tags:
+        - Pageviews data
+      description: >
+        List pageviews data sub-apis.
+
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Stable)
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: The queriable sub-items
+          schema:
+            $ref: '#/definitions/listing'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-monitor: false
+
+  /{module:pageviews}/per-article/{project}/{access}/{agent}/{article}/{granularity}/{start}/{end}:
+    get:
+      tags:
+        - Pageviews data
+        - per-article
+      description: >
+        List pageviews for a given article in a project, over a given time period.
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: project
+          in: path
+          description: Name of the project
+          type: string
+          required: true
+        - name: access
+          in: path
+          description: Type of access method [all-access|desktop|mobile-app|mobile-web]
+          type: string
+          required: true
+        - name: agent
+          in: path
+          description: Type of user agent [all-agents|user|spider]
+          type: string
+          required: true
+        - name: article
+          in: path
+          description: Name of the article for which to get data
+          type: string
+          required: true
+        - name: granularity
+          in: path
+          description: Time granularity of the response data [hourly|daily]
+          type: string
+          required: true
+        - name: start
+          in: path
+          description: Start timestamp in YYYYMMDDHH format
+          type: string
+          required: true
+        - name: end
+          in: path
+          description: End timestamp in YYYYMMDDHH format
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The list of values
+          schema:
+            $ref: '#/definitions/article'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-backend-request:
```

Comment:
This code requires a rebase with `wikimedia/restbase` master, because `x-backend-request` stanza was removed and replaced with `x-request-handler`.
See [docs](https://github.com/wikimedia/restbase/blob/master/doc/HandlerConfiguration.md) for examples of current handler configuration options.